mimecast/help.md

```diff
@@ -1,5 +1,30 @@
 # Description
 
+Manage Mimecast Services
+
+# Key Features
+
+Manage blocked email domains, manage block list of browsing domains
+
+# Requirements
+
+* API Username and Password
+* API Key
+
+# Documentation
+
+* https://www.mimecast.com/tech-connect/documentation/api-overview
+
+## Setup
+
+No special instructions for setting up this plugin
+
+## Technical Details
+
+No notable technical requirements to highlight
+
+## About
+
```

Comment:
Help markdown is not properly formatted here - it looks like there are some duplicate headings. 
Top level headings should be:
Description
Key Features
Requirements
Documentation
Version History
Links

Reply:
it looks like you have appended some stuff to the top of the markdown that should not be there